### DISCUSSION TITLE
Saving / Exporting Settings

### DISCUSSION THREAD

**Discussion by geoff-maddock** [noisedeck/noisedeck-docs#8] [Ideas]
I'd love to save my settings and be able to return to that state or share with others.

**Comment by jesshewitt** [comment]
We do plan to implement this soon!

**Reply by franck-friends** [reply]
@jesshewitt 
This feature has been implemented quite a while ago but today I've noticed that all my past user settings have suddenly disappeared.
Are those settings saved in your DB or are they stored in the browser?

**Reply by jesshewitt** [reply]
They are stored in the browser only, for now.
